blueprint/core/blueprint_GenericEditor.cpp

```diff
@@ -27,7 +31,11 @@ namespace blueprint
 
         // Setup the ReactApplicationRoot callbacks and evaluate the supplied JS code/bundle
         registerAppRootCallbacks();
+        harness.watch(bundleFile);
```

Comment:
Yea, can do. I was thinking it would make sense to just _not_ use a HotReloading Harness in release mode, but that puts more work on the user. Plenty easy to add that case in here.